lib/stdlib/src/erl_lint.erl

```diff
@@ -135,7 +135,9 @@ value_option(Flag, Default, On, OnVal, Off, OffVal, Opts) ->
                types = dict:new()               %Type definitions
                    :: dict:dict(ta(), #typeinfo{}),
                exp_types=gb_sets:empty()        %Exported types
-                   :: gb_sets:set(ta())
+                   :: gb_sets:set(ta()),
+               remotes = dict:new()             %Referenced modules
+                   :: dict:dict(module(), term())
```

Comment:
Surely the "value" here is not _any_ `term()`, but something more specific than that, right?